src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -299,6 +299,16 @@ The first part date {1} should be lesser than or equal to the second part date {
     <value>The CR code '{0}' is not supported.</value>
     <comment>{0} VR Code.</comment>
   </data>
+  <data name="Forbidden" xml:space="preserve">
+    <value>Authorization failed.</value>
+  </data>
+  <data name="ErrorValidatingRoles" xml:space="preserve">
+    <value>Error validating roles:
+{0}</value>
```

Comment:
Just confirming: is the newline intended?

Reply:
That's wired, let me double check.

Reply:
yes... and the code in main is same, let me check with Brandon.
https://github.com/microsoft/dicom-server/blob/401bcc5ff16fbeb835b7ddddff1070c4243c9f78/src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx#L309

Reply:
it turns out coming from https://github.com/microsoft/fhir-server/blob/5f1b6365ef3484bed48581adeba3878309136381/src/Microsoft.Health.Fhir.Core/Resources.resx#L195